MibTeX/src/de/mibtex/export/ExportHTML.java

```diff
@@ -207,9 +207,13 @@ private String getHTMLAuthor(BibtexEntry entry) {
 	public static String getHTMLTitle(BibtexEntry entry) {
 		String htmlTitle = "<a href=\"" + entry.getRelativePDFPath() + "\">";
 		if (entry.getPDFPath().exists())
-			htmlTitle += entry.title + "</a>";
+		{
```

Comment:
Line break before bracket is not defined in the auto formatter an inconsistent with the rest of the project

Reply:
removed line breaks